01-numpy.md

```diff
@@ -544,38 +544,34 @@ Neither result seems particularly likely,
 so either there's a mistake in our calculations
 or something is wrong with our data.
 
-It's very common to create an **alias** for a library when importing it
-in order to reduce the amount of typing we have to do.
-Here are our three plots side by side using aliases for `numpy` and `pyplot`:
+Here are our three plots side by side:
```

Comment:
I appreciate that the concept of making aliases is a bit unnecessary and unrelated to other pedagogical goals at this point, but it's not a point that has to be understood deeply (or even has any real deep meaning); it's enough to say that you can make shortcuts to save on typing. Perhaps we should just import with an alias from the very beginning, rather than waiting until now to do it?

Briefly, my justification for wanting to keep `np` and `plt` in general is that this is the standard way to write numpy / matplotlib code, which means our code will look like what students see when they google for answers to their specific questions. Saving typing is a non-trivial gain as well, as we don't want to make students type so much that they resort to `%pylab inline` or `from numpy import *`, which is sometimes recommended for the purpose of saving typing.